nnthanh/iercapstone
# [WIP] Initial Work Ajax Request HopDongLayMau

Ajax request HopDongPhanTich posting (DRAFT)
To test:
Run the project and access /Administrator/Request/Create
Input data (including the MauLayHienTruong data) and click submit
Access /Administrator/Request/ and /Administrator/MauLayHienTruong to verify data inserted into the db
